lib/still/compiler/incremental/node.ex

```diff
@@ -30,26 +31,56 @@ defmodule Still.Compiler.Incremental.Node do
 
   @default_compilation_timeout :infinity
 
+  @impl true
   def start_link(file: file) do
     GenServer.start_link(__MODULE__, %{file: file}, name: file |> String.to_atom())
   end
 
+  @doc """
+  Compiles the file mapped by the `Node` with the given PID.
+
+  This PID can be obtained from `Still.Compiler.Incremental.Registry`.
+
+  For difference between compilation and renderisation see
+  `Still.Compiler.File`.
+  """
   def compile(pid) do
     GenServer.call(pid, :compile, compilation_timeout())
   end
 
+  @doc """
+  Renders the file mapped by the `Node` with the given PID.
+
+  This PID can be obtained from `Still.Compiler.Incremental.Registry`.
+
+  For difference between compilation and renderisation see
+  `Still.Compiler.File`.
+  """
   def render(pid, data, subscriber \\ nil) do
     GenServer.call(pid, {:render, data, subscriber}, compilation_timeout())
   end
 
+  @doc """
+  TODO
```

Comment:
@gabrielpoca can you make a suggestion here?

Reply:
```suggestion
  Adds a file to the list of files this process is subscribed to.
```